applications/machine_learning/Kconfig.sysbuild

```diff
@@ -20,6 +20,9 @@ config NETCORE_APP_UPDATE
 config PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY
 	default y if BOARD_NRF5340DK_NRF5340_CPUAPP
 
+config MCUBOOT_MODE_OVERWRITE_ONLY
```

Comment:
@nordicjm so how do you default choices for boards? because this didn't work and making it a choice didn't seem to do what I intended.

Reply:
it needs to be in the choice part i.e.
```
choice MCUBOOT_MODE
default MCUBOOT_MODE_OVERWRITE_ONLY if SOC_SERIES_NRF53
endchoice
```

Reply:
Well as I tried to explain above I've tried this:

```
warning: default on the choice symbol MCUBOOT_MODE_OVERWRITE_ONLY (defined at /applications/machine_learning/Kconfig.sysbuild:23, /nrf/sysbuild/Kconfig.mcuboot:42) will have no effect, as defaults do not affect choice symbols
```

I can make a change directly to the Kconfig but that doesn't make sense either.

Reply:
I don't know what you mean, I just put this in `Kconfig.sysbuild` and it works perfectly??
```
choice MCUBOOT_MODE
        default MCUBOOT_MODE_OVERWRITE_ONLY if SOC_SERIES_NRF53X
endchoice
```

Reply:
I think I was just stupid.